config/application.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require_relative 'boot'
+require_relative '../lib/logging/json_formatter.rb'
```

Comment:
@whereismyjetpack if you make the file location `lib/json_formatter.rb` than this can be written as `require 'json_formatter'`